app/routes.py

```
@@ -0,0 +1,40 @@
+            request.old_commit,
+            request.new_commit,
+            repo_name
+        )
+        
+        return ComparisonResponse(
+            status="success",
+            message="Comparison completed successfully",
+            diff_file=f"updates/{result.repo_name}-{result.timestamp.strftime('%Y-%m-%d')}-open-api-diff.md"
+        )
+        
+    except Exception as e:
```

Ruff (0.8.2)

Do not catch blind exception: `Exception`

(BLE001)